### PR TITLE
Add command line tool to query / upload to terrarium

### PR DESCRIPTION
**Pull request by adcharre**
This is currently in progress but I thought worth creating the PR now so people can see it.

Initially focusing on publishing modules and releases I've also added a command to query the container dependencies.

### CODE REVIEW

File: tools/cli/cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
```
Comment (odelanno) [inline tools/cli/cmd/root.go:2]: Please fill in the copyright

File: tools/cli/main.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/*
```
Comment (odelanno) [inline tools/cli/main.go:1]: copyright.

**Review by odelanno — CHANGES_REQUESTED** [review]

File: tools/cli/cmd/root.go (hunk 1 above)
Comment (adcharre) [reply tools/cli/cmd/root.go]: Removed template copyright comment - already have LICENSE in the root of the repo

File: tools/cli/main.go (hunk 2 above)
Comment (adcharre) [reply tools/cli/main.go]: Removed template copyright comment - already have LICENSE in the root of the repo

**Review by odelanno — APPROVED** [review]

**Pull request merged**